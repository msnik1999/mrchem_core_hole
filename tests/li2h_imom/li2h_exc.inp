# vim:syntax=sh:

world_prec = 1.0e-3                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n

MPI {
  numerically_exact = true          # Guarantee identical results in MPI
}

Molecule {
  charge = 1
$coords
Li    -5.46737452    2.65708721    0.38412463
Li     2.48697407    1.97066310   -0.29143356
H     -0.09374931   -1.62164958   -3.76992803
$end
}

WaveFunction {
  method = PBE                      # Wave function method (HF or DFT)
}

OrbitalOccupancies {
  occupancies = "1 1 0"
}

SCF {
  run = true
  guess_type = mw                   # Type of initial guess: none, mw, gto
  deltascf_method = IMOM
}