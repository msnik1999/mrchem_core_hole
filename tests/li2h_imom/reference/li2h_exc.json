{
  "input": {
    "constants": {
      "N_a": 6.02214076e+23,
      "angstrom2bohrs": 1.8897261246257702,
      "boltzmann_constant": 1.380649e-23,
      "dipmom_au2debye": 2.5417464739297717,
      "e0": 8.8541878128e-12,
      "electron_g_factor": -2.00231930436256,
      "elementary_charge": 1.602176634e-19,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084,
      "meter2bohr": 18897261246.2577
    },
    "geom_opt": {
      "init_step_size": -0.5,
      "max_force_component": 0.005,
      "max_history_length": 10,
      "max_iter": 100,
      "minimal_step_size": 0.01,
      "run": false,
      "subspace_tolerance": 0.001,
      "use_previous_guess": false
    },
    "molecule": {
      "charge": 1,
      "coords": [
        {
          "atom": "li",
          "r_rms": 4.0992133976e-05,
          "xyz": [
            -5.46737452,
            2.65708721,
            0.38412463
          ]
        },
        {
          "atom": "li",
          "r_rms": 4.0992133976e-05,
          "xyz": [
            2.48697407,
            1.9706631,
            -0.29143356
          ]
        },
        {
          "atom": "h",
          "r_rms": 2.6569547399e-05,
          "xyz": [
            -0.09374931,
            -1.62164958,
            -3.76992803
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "omp_threads": -1,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "imom",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "nuclear_model": "point_like",
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "None",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (PBE)",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "mw",
        "zeta": 0
      },
      "occupancies": [
        {
          "occupancy": [
            1.0,
            0.0
          ],
          "orbital": 1
        }
      ],
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "deltascf_method": "IMOM",
        "energy_thrs": -1.0,
        "environment": "None",
        "external_field": "None",
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 0.001,
        "helmholtz_prec": -1.0,
        "kain": 5,
        "localize": false,
        "max_iter": 100,
        "method": "DFT (PBE)",
        "orbital_thrs": 0.01,
        "relativity": "None",
        "rotation": 0,
        "start_prec": 0.001
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -1.3956707147894085,
        2.047468923584971,
        -0.2119882646732089
      ],
      "charge": 1,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 4.9985802763310305,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -2.4527956056764655,
            4.351903205370137,
            -0.17474835275030198
          ],
          "vector_el": [
            6.582155054323275,
            -7.909698144629003,
            3.317106467249352
          ],
          "vector_nuc": [
            -9.03495065999974,
            12.26160134999914,
            -3.491854819999654
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Li",
          "xyz": [
            -5.46737452,
            2.65708721,
            0.38412463
          ]
        },
        {
          "symbol": "Li",
          "xyz": [
            2.48697407,
            1.9706631,
            -0.29143356
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -0.09374931,
            -1.62164958,
            -3.76992803
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -3.275280997260719,
          -2.3259664241828286,
          -0.5843179998112323
        ],
        "occupation": [
          1.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -9.095849845248841
      },
      "orbital_positions": [
        [
          2.4889564561238546,
          1.9719234020454615,
          -0.2901967280717931
        ],
        [
          -5.4662363142960295,
          2.6570241763369,
          0.3840467611532139
        ],
        [
          0.9306805590724647,
          0.31186319495486986,
          -1.8975016307419938
        ]
      ],
      "scf_energy": {
        "E_ee": 6.306475329573725,
        "E_eext": 0.0,
        "E_el": -14.6261957555,
        "E_en": -30.4471911355338,
        "E_kin": 12.36586302907365,
        "E_next": 0.0,
        "E_nn": 2.03010184805887,
        "E_nuc": 2.03010184805887,
        "E_tot": -12.59609390744113,
        "E_x": 0.0,
        "E_xc": -2.8513429786135758,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 8,
      "routine": "mrchem.x",
      "total_cores": 8,
      "version": "1.2.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 6.233158389316568,
        "E_eext": 0.0,
        "E_el": -14.2655874343218,
        "E_en": -28.952858149286865,
        "E_kin": 11.335353768812364,
        "E_next": 0.0,
        "E_nn": 2.03010184805887,
        "E_nuc": 2.03010184805887,
        "E_tot": -12.23548558626293,
        "E_x": 0.0,
        "E_xc": -2.881241443163868,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 6.051861527813008,
              "E_eext": 0.0,
              "E_el": -14.415838368728469,
              "E_en": -29.031703220846644,
              "E_kin": 11.367499048137933,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.3857365206696,
              "E_x": 0.0,
              "E_xc": -2.803495723832765,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.3857365206696,
            "energy_update": 0.15025093440666915,
            "mo_residual": 0.22719511992818403,
            "wall_time": 1.665127
          },
          {
            "energy_terms": {
              "E_ee": 6.1430552870483215,
              "E_eext": 0.0,
              "E_el": -14.517390957765501,
              "E_en": -29.650350402733793,
              "E_kin": 11.814551635302355,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.487289109706632,
              "E_x": 0.0,
              "E_xc": -2.8246474773823853,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.487289109706632,
            "energy_update": 0.10155258903703235,
            "mo_residual": 0.15498991767288342,
            "wall_time": 1.304980209
          },
          {
            "energy_terms": {
              "E_ee": 6.205533920782777,
              "E_eext": 0.0,
              "E_el": -14.563985694377278,
              "E_en": -29.991032394163753,
              "E_kin": 12.058774476012834,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.533883846318409,
              "E_x": 0.0,
              "E_xc": -2.837261697009138,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.533883846318409,
            "energy_update": 0.04659473661177671,
            "mo_residual": 0.11813237005059271,
            "wall_time": 1.322150084
          },
          {
            "energy_terms": {
              "E_ee": 6.398169391112515,
              "E_eext": 0.0,
              "E_el": -14.611928199237823,
              "E_en": -30.65164806209109,
              "E_kin": 12.50689031499396,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.581826351178954,
              "E_x": 0.0,
              "E_xc": -2.865339843253207,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.581826351178954,
            "energy_update": 0.0479425048605453,
            "mo_residual": 0.3442647236426666,
            "wall_time": 1.57771525
          },
          {
            "energy_terms": {
              "E_ee": 6.3008022924877025,
              "E_eext": 0.0,
              "E_el": -14.616553378682381,
              "E_en": -30.363570634614398,
              "E_kin": 12.293671585263002,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.586451530623512,
              "E_x": 0.0,
              "E_xc": -2.847456621818688,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.586451530623512,
            "energy_update": 0.004625179444557759,
            "mo_residual": 0.11714358991046543,
            "wall_time": 1.473116667
          },
          {
            "energy_terms": {
              "E_ee": 6.30792465491902,
              "E_eext": 0.0,
              "E_el": -14.623147387168297,
              "E_en": -30.384433557523387,
              "E_kin": 12.302270994467692,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.593045539109427,
              "E_x": 0.0,
              "E_xc": -2.8489094790316205,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.593045539109427,
            "energy_update": 0.006594008485915737,
            "mo_residual": 0.07715302224759014,
            "wall_time": 1.495685542
          },
          {
            "energy_terms": {
              "E_ee": 6.328010948454974,
              "E_eext": 0.0,
              "E_el": -14.625216481827374,
              "E_en": -30.480138031404227,
              "E_kin": 12.382677906221318,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.595114633768505,
              "E_x": 0.0,
              "E_xc": -2.855767305099439,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.595114633768505,
            "energy_update": 0.002069094659077564,
            "mo_residual": 0.04071518187656166,
            "wall_time": 1.877093292
          },
          {
            "energy_terms": {
              "E_ee": 6.3146425238894,
              "E_eext": 0.0,
              "E_el": -14.625514708727625,
              "E_en": -30.461342192472557,
              "E_kin": 12.37603469696528,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.595412860668755,
              "E_x": 0.0,
              "E_xc": -2.854849737109749,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.595412860668755,
            "energy_update": 0.0002982269002504978,
            "mo_residual": 0.024249810788043204,
            "wall_time": 1.527157083
          },
          {
            "energy_terms": {
              "E_ee": 6.308827483512665,
              "E_eext": 0.0,
              "E_el": -14.626262224862344,
              "E_en": -30.456034640725406,
              "E_kin": 12.37336279278835,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.596160376803475,
              "E_x": 0.0,
              "E_xc": -2.8524178604379538,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.596160376803475,
            "energy_update": 0.0007475161347194614,
            "mo_residual": 0.03308895675846697,
            "wall_time": 1.538616125
          },
          {
            "energy_terms": {
              "E_ee": 6.30604909195298,
              "E_eext": 0.0,
              "E_el": -14.626052207155217,
              "E_en": -30.436443459743504,
              "E_kin": 12.355234833819,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.595950359096348,
              "E_x": 0.0,
              "E_xc": -2.8508926731836937,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.595950359096348,
            "energy_update": 0.00021001770712736345,
            "mo_residual": 0.014453147861862435,
            "wall_time": 1.530274375
          },
          {
            "energy_terms": {
              "E_ee": 6.3055365819142555,
              "E_eext": 0.0,
              "E_el": -14.626229615133417,
              "E_en": -30.44219706231865,
              "E_kin": 12.361148537441428,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.596127767074547,
              "E_x": 0.0,
              "E_xc": -2.8507176721704464,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.596127767074547,
            "energy_update": 0.00017740797819953968,
            "mo_residual": 0.010858352373482727,
            "wall_time": 1.695477208
          },
          {
            "energy_terms": {
              "E_ee": 6.306475329573725,
              "E_eext": 0.0,
              "E_el": -14.6261957555,
              "E_en": -30.4471911355338,
              "E_kin": 12.36586302907365,
              "E_next": 0.0,
              "E_nn": 2.03010184805887,
              "E_nuc": 2.03010184805887,
              "E_tot": -12.59609390744113,
              "E_x": 0.0,
              "E_xc": -2.8513429786135758,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -12.59609390744113,
            "energy_update": 3.385963341706599e-05,
            "mo_residual": 0.0034640266944375524,
            "wall_time": 1.793126458
          }
        ],
        "wall_time": 18.802563208
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}