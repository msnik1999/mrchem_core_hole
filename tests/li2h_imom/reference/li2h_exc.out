***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         mom_merged                               ***
***         Git commit hash    aec5c05c74a8084661ef-dirty               ***
***         Git commit author  Niklas GÃ¶llmann                         ***
***         Git commit date    Mon Jun 3 10:21:30 2024 +0200            ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               8
 Total cores             :                                               8

---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.6.0-alpha
 Git branch            : mom
 Git commit hash       : 9ebc1da6290a65a252e6
 Git commit author     : Laura Ratcliff
 Git commit date       : Wed Jun 14 14:11:39 2023 +0200

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (8 threads)

---------------------------------------------------------------------------



===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================



***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************


===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :       -5.467375        2.657087        0.384125
    1      Li            :        2.486974        1.970663       -0.291434
    2       H            :       -0.093749       -1.621650       -3.769928
---------------------------------------------------------------------------
 Center of mass          :       -1.395671        2.047469       -0.211988
===========================================================================



***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************


Warning: guess_orbitals(), line 405: Occupancies for alpha and beta orbitals are added together for restricted calculations
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Project MW molecular orbitals
 Precision               : 1.00000e-03
 Restricted              : True
 MO file                 : initial_guess/phi_p_scf
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                           2.500
 Beta electrons          :                                           2.500
 Total electrons         :                                           5.000
---------------------------------------------------------------------------
    n     Occ Spin       :                                            Norm
---------------------------------------------------------------------------
    0    2.00    p       :                              9.999520240476e-01
    1    1.00    p       :                              9.999996902336e-01
    2    2.00    p       :                              9.999999999802e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE)
 Relativity              : None
 Environment             : None
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 11.335353768812
 E-N energy              :            (au)                -28.952858149287
 Coulomb energy          :            (au)                  6.233158389317
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -2.881241443164
 N-N energy              :            (au)                  2.030101848059
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -14.265587434322
 Nuclear energy          :            (au)                  2.030101848059
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.223548558626e+01
                         :      (kcal/mol)             -7.677883125142e+03
                         :        (kJ/mol)             -3.212426299559e+04
                         :            (eV)             -3.329445241950e+02
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n     Occ Spin       :                                         Epsilon
---------------------------------------------------------------------------
    0    2.00    p       :            (au)                 -3.567173111761
    1    1.00    p       :            (au)                 -2.320683561023
    2    2.00    p       :            (au)                 -0.311443255986
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -10.077916296519
===========================================================================



***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE)
 Relativity              : None
 Environment             : None
 External fields         : None
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          1.732051e+00         -12.235485586263         -1.223549e+01
    1          2.271951e-01         -12.385736520670         -1.502509e-01
    2          1.549899e-01         -12.487289109707         -1.015526e-01
    3          1.181324e-01         -12.533883846318         -4.659474e-02
    4          3.442647e-01         -12.581826351179         -4.794250e-02
    5          1.171436e-01         -12.586451530624         -4.625179e-03
    6          7.715302e-02         -12.593045539109         -6.594008e-03
    7          4.071518e-02         -12.595114633769         -2.069095e-03
    8          2.424981e-02         -12.595412860669         -2.982269e-04
    9          3.308896e-02         -12.596160376803         -7.475161e-04
   10          1.445315e-02         -12.595950359096          2.100177e-04
   11          1.085835e-02         -12.596127767075         -1.774080e-04
   12          3.464027e-03         -12.596093907441          3.385963e-05
---------------------------------------------------------------------------
                      SCF converged in 12 iterations!
===========================================================================



***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************


===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :       -5.467375        2.657087        0.384125
    1      Li            :        2.486974        1.970663       -0.291434
    2       H            :       -0.093749       -1.621650       -3.769928
---------------------------------------------------------------------------
 Center of mass          :       -1.395671        2.047469       -0.211988
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 12.365863029074
 E-N energy              :            (au)                -30.447191135534
 Coulomb energy          :            (au)                  6.306475329574
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                 -2.851342978614
 N-N energy              :            (au)                  2.030101848059
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -14.626195755500
 Nuclear energy          :            (au)                  2.030101848059
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.259609390744e+01
                         :      (kcal/mol)             -7.904168263107e+03
                         :        (kJ/mol)             -3.307104001284e+04
                         :            (eV)             -3.427571765061e+02
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n     Occ Spin       :                                         Epsilon
---------------------------------------------------------------------------
    0    1.00    p       :            (au)                 -3.275280997261
    1    2.00    p       :            (au)                 -2.325966424183
    2    2.00    p       :            (au)                 -0.584317999811
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -9.095849845249
===========================================================================


===========================================================================
                             Orbital Positions
---------------------------------------------------------------------------
Index                    :               x               y               z
---------------------------------------------------------------------------
    0                    :        2.488956        1.971923       -0.290197
    1                    :       -5.466236        2.657024        0.384047
    2                    :        0.930681        0.311863       -1.897502
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        6.582155       -7.909698        3.317106
 Magnitude               :            (au)                       10.811627
                         :         (Debye)                       27.480416
---------------------------------------------------------------------------
 Nuclear vector          :       -9.034951       12.261601       -3.491855
 Magnitude               :            (au)                       15.625948
                         :         (Debye)                       39.717198
---------------------------------------------------------------------------
 Total vector            :       -2.452796        4.351903       -0.174748
 Magnitude               :            (au)                        4.998580
                         :         (Debye)                       12.705124
===========================================================================




***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 19s                       ***
***                                                                     ***
***************************************************************************